docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -171,6 +182,169 @@ fails.
+
+.. code-block:: none
+
+    ipa-replica-install
+    WARNING: conflicting time&date synchronization service 'chronyd' will
+    be disabled in favor of ntpd
+
+    ipa         : ERROR    Reverse DNS resolution of address 192.0.2.5 (ipa-server.example.com) failed. Clients may not function properly. Please check your DNS setup. (Note that this check queries IPA DNS directly and ignores /etc/hosts.)
+    Continue? [no]: yes
+    Run connection check to master
+    Connection check OK
+    Configuring NTP daemon (ntpd)
+      [1/4]: stopping ntpd
+      [2/4]: writing configuration
+      [3/4]: configuring ntpd to start on boot
+      [4/4]: starting ntpd
+    Done configuring NTP daemon (ntpd).
+    Configuring directory server (dirsrv). Estimated time: 1 minute
+
+
+.. tip::
+    It's ok if ipa-replica-install gives error about "Reverse DNS resolution".
+    But if the installation fails with the following.
+
+    .. code-block:: none
+
+        [error] RuntimeError: failed to create ds instance Command '/usr/sbin/setup-ds --silent --logfile - -f /tmp/tmpuhBVEm' returned non-zero exit status 1
+
+    It may be that you either misconfigured /etc/hosts or need to add a reverse
+    DNS entry for the replica.
+
+Setup DNS on replica
+^^^^^^^^^^^^^^^^^^^^
+
+It's a good idea to setup DNS on at least on of your replicas and use it as a
+failover DNS on the clients. This can only be done once the host is setup as a
+replica.
+
+Configure the same forwarders as your master ipa server:
+
+.. code-block:: none
+    :caption: Example installation of IPA DNS
+
+    ipa-dns-install
+
+    The log file for this installation can be found in /var/log/ipaserver-install.log
+    ==============================================================================
+    This program will setup DNS for the FreeIPA Server.
+
+    This includes:
+      * Configure DNS (bind)
+      * Configure SoftHSM (required by DNSSEC)
+      * Configure ipa-dnskeysyncd (required by DNSSEC)
+
+    NOTE: DNSSEC zone signing is not enabled by default
+
+
+    To accept the default shown in brackets, press the Enter key.
+
+    Do you want to configure DNS forwarders? [yes]: 
+    Following DNS servers are configured in /etc/resolv.conf: 198.51.100.53, 192.0.2.66, 192.0.2.67
+    Do you want to configure these servers as DNS forwarders? [yes]: no
+    Enter an IP address for a DNS forwarder, or press Enter to skip: 192.0.2.66
+    DNS forwarder 192.0.2.66 added. You may add another.
+    Enter an IP address for a DNS forwarder, or press Enter to skip: 192.0.2.67
+    DNS forwarder 192.0.2.67 added. You may add another.
+    Enter an IP address for a DNS forwarder, or press Enter to skip: 
+    Checking DNS forwarders, please wait ...
+    Do you want to search for missing reverse zones? [yes]: 
+
+    The following operations may take some minutes to complete.
+    Please wait until the prompt is returned.
+
+If the installation fails, follow the instructions in the troubleshooting section
```

Comment:
As @frznlogic has already mentioned, shouldn't all the above be part of the "Troubleshooting"? This is troubleshooting to me: issue description -> possible cause(s) and proposed solution(s)

Reply:
Yes, moving this to the troubleshooting section.